web/packages/extension/assets/options.html

```diff
@@ -71,6 +53,37 @@
                 />
                 <label for="player_version">Flash player version number to emulate (range 1-32)</label>
             </div>
+            <details>
+                <summary>Advanced Options</summary>
+                <div class="options">
+                    <div class="option select">
+                        <select id="log_level">
+                            <option value="info">Info</option>
+                            <option value="warn">Warning</option>
+                            <option value="error">Error</option>
+                        </select>
+                        <label for="log_level">Log level</label>
+                    </div>
+                    <div class="option checkbox">
+                        <input type="checkbox" id="ignore_optout" />
+                        <label for="ignore_optout">Ignore website compatibility warnings</label>
+                    </div>
+                    <div class="option select">
+                        <select id="force_renderer">
+                            <option value="">None</option>
+                            <option value="webgpu">WebGPU</option>
+                            <option value="wgpu-webgl">Wgpu (via WebGL)</option>
+                            <option value="webgl">WebGL</option>
+                            <option value="canvas">Canvas2D</option>
+                        </select>
+                        <label for="force_renderer">Force renderer</label>
```

Comment:
Would it make sense to inform the user somehow of the exclusivity of "Preferred renderer" and "Force renderer", apart from "Preferred renderer" being disabled if "Force renderer" is set?

Reply:
Like, maybe "Force renderer (exclusive with "Preferred renderer")?

That said, "Force renderer" is an "Advanced option", which helps a lot.